packages/transport-bluetooth/src/client/types.ts

```diff
@@ -0,0 +1,46 @@
+import type { TypedEmitter } from '@trezor/utils';
+
+export interface BluetoothDevice {
+    name: string;
+    id: string;
+    data: number[]; // advertisement data bytes
+    connected: boolean;
+    timestamp?: number; // last know activity from the device (discovery, advertisements)
+    paired?: boolean; // known (linux, windows), unknown (macos)
+}
+
+// IpcApi related types
+export type DeviceConnectionStatus = { uuid: string } & (
+    | { type: 'pairing'; pin?: string }
+    | { type: 'paired' }
+    | { type: 'connecting' }
+    | { type: 'connected' }
+);
+
+type Success<P> = P extends unknown ? { success: true } : { success: true; payload: P };
+type Failure = { success: false; error: string };
+type IpcResponse<P = unknown> = Success<P> | Failure;
+
+export interface BluetoothIpcEvents {
+    'adapter-event': boolean;
+    'device-list-update': BluetoothDevice[];
+    'device-connection-status': DeviceConnectionStatus;
+}
+
+type TypedManagerEvents = TypedEmitter<BluetoothIpcEvents>;
+
+export interface BluetoothIpcState {
+    knownDevices: BluetoothDevice[];
```

Comment:
knownDevices? pairedDevices?

it's named `pairedDevices` in bluetooth reducer, but in `TrezorConnect` it will be called `knownDevices`

Reply:
should we pass `bluetoothEnabled` flag here and basing on that start the binary and scanning process or not.
see implementation in  [initBluetoothThunk.ts](https://github.com/trezor/trezor-suite/commit/0090e10d473b6a02a527d2364dd719ec03fd3778#diff-6f48d4ff395f971f15bdbe519ea21a792a73e28861859de5e48310757dbd2dfe)

Reply:
knownDevices it is, discussed IRL